foreign_cc/private/framework/toolchains/freebsd_commands.bzl

```diff
@@ -83,7 +83,7 @@ if [ -d "$1" ]; then
   for file in ${files[@]+"${files[@]}"}; do
     local backup=$(mktemp)
     touch -r "${file}" "${backup}"
-    sed -i '' -e 's@'"$2"'@'"$3"'@g' "${file}"
+    /usr/bin/sed -i '' -e 's@'"$2"'@'"$3"'@g' "${file}"
```

Comment:
Is this definitely correct on BSD? I don't have a bsd system to be able to test this on - is there some docs on the net that you can point to to back this up?

Reply:
I couldn't find any documentation but I just created a FreeBSD 12.4 virtual machine and it's /usr/bin/sed.

```
root@freebsd:~ # which sed
/usr/bin/sed
root@freebsd:~ # uname -a
FreeBSD freebsd 12.4-RELEASE-p1 FreeBSD 12.4-RELEASE-p1 GENERIC  amd64
```